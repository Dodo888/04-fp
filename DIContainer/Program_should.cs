using System.IO;
using ApprovalTests;
using ApprovalTests.Reporters;
using FakeItEasy;
using NUnit.Framework;

namespace DIContainer
{
    [TestFixture]
	[UseReporter(typeof(DiffReporter))]
    public class Program_should
    {
	    [Test]
	    public void show_help()
	    {
		    var output = new StringWriter();
		    
			Program.Run(output, "help");

		    Approvals.Verify(output.ToString().Replace("\r", ""));
	    }

		[Test]
		public void show_detailed_help()
		{
			var output = new StringWriter();

			Program.Run(output, "help help");

			Approvals.Verify(output.ToString().Replace("\r", ""));
		}
    }
}