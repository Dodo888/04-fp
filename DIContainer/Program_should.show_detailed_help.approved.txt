Help command:
  Help -- lists available commands
  Help <commandName> -- display help for specified command 